src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/repository/PrisonIncentiveLevelRepositoryTest.kt

```diff
@@ -65,10 +65,10 @@ class PrisonIncentiveLevelRepositoryTest : TestBase() {
       levelCode = "STD",
       prisonId = "MDI",
 
-      remandTransferLimitInPence = 2500,
-      remandSpendLimitInPence = 25000,
-      convictedTransferLimitInPence = 500,
-      convictedSpendLimitInPence = 5000,
+      remandTransferLimitInPence = 2750,
```

Comment:
Any reason why you didn't use the `_` to delimit the pence in the tests as well like in the `spendLimitPolicy` value? I quite like it and it helps readability.

Reply:
Maybe to avoid possible confusion with the values in the JSON strings?

Reply:
no good reason :) will replace in next pr